src/getAppSyncConfig.js

```diff
@@ -13,6 +13,11 @@ const directLambdaMappingTemplates = {
   response: directLambdaResponse,
 };
 
+const mappingTemplateType = {
+  mappingTemplate: 'mappingTemplate',
+  functionConfiguration: 'functionConfiguration',
+};
+
```

Comment:
It all LGTM!
Is there any added value to this "mapping"?
We already use hardcoded strings for `request` and `response` for `type`

possibly, we could use constants everywhere if we wanted to improve it slightly.

Reply:
Thanks for your questions @bboure. I think we should store this "mapping" to another location and refactor some hardcoded strings into it as I recently added.